webpack.config.js

```diff
@@ -64,11 +64,13 @@ module.exports = {
     ],
   },
   devServer: {
```

Comment:
does this map over to `mode=development` in the webpack build command?

Reply:
Yes, these are the dev server options